docs/datasets.md

```diff
@@ -29,47 +29,69 @@ np_datasets = tfds.as_numpy(datasets)
 ## All Datasets
 
 * [`audio`](#audio)
```

Comment:
This file is auto-generated.

You should change the generator instead: scripts/document_datasets.py

Reply:
@cyfra Should I make a new PR for that and close this one or make changes here only?

Reply:
Thanks for your contribution. I think https://github.com/tensorflow/datasets/pull/145 is already accomplishing this so we could probably close this one.

Reply:
okay :+1: